packages/app-desktop/gui/NoteList/utils/useRefocusOnDeletion.ts

```
@@ -0,0 +1,19 @@
+import { useEffect } from 'react';
+import usePrevious from '@joplin/lib/hooks/usePrevious';
```

ESLint

[error] Expected linebreaks to be 'LF' but found 'CRLF'.

(linebreak-style)